### ISSUE TITLE
Add rule `no-goto-without-base`

### ISSUE THREAD

**Issue by marekdedic** [sveltejs/eslint-plugin-svelte#675]
### Motivation

In SvelteKit, the `goto` function is used for internal navigation. That means, that you basically almost always want to prefix the url with the `base` path.

### Description

Add a rule that would trigger on any `goto` without a `base` path. However, there are quite some edge cases

### Examples

```svelte
<script>
  import { base } from "$app/paths";
  import { base as whatever } from "$app/paths";
  import { goto } from "$app/navigation";

  <!-- ✓ GOOD -->
  goto(base + "/foo");
  goto(`${base}/foo`)
  goto(whatever + "/foo");
  goto(`${whatever}/foo`)
  goto("https://absolute.url");

  <!-- ✗ BAD -->
  goto("/foo");
  goto("foo/" + base);
  goto(`foo/${base}`);
</svelte>
```

### Additional comments

Is this plugin the right place to put SvelteKit rules?

**Comment by ota-meshi** [comment]
Thank you for posting proposed rule!

> Is this plugin the right place to put SvelteKit rules?

I think it's good that the rules regarding what svelte official provides are provided by this plugin.
